src/main/org/tvrenamer/controller/util/FileUtilities.java

```diff
@@ -84,8 +84,9 @@ private static boolean doAction(File source, File dest, ProgressObserver observe
                 IOUtilities.closeQuietly(fis);
             }
 
-            if (ok && deleteOnSuccess)
+            if (ok && deleteOnSuccess) {
```

Comment:
I cannot 👍 this enough so 👍*∞. The fact that java allow `if` statements without braces is terrible